src/extensions/auto-refresh/bootstrap-table-auto-refresh.js

```diff
@@ -0,0 +1,85 @@
+/* global jQuery:true */
+
+/**
+ * @author: Alec Fenichel
+ * @webSite: https://fenichelar.com
+ * @version: v1.0.0
+ */
+
+(function ($) {
+
+  'use strict';
+
+  $.extend($.fn.bootstrapTable.defaults, {
+    autoRefresh: false,
+    autoRefreshInterval: 60,
+    autoRefreshSilent: true,
+    autoRefreshStatus: true,
+
+    autoRefreshFunction: null
+  });
+
+  $.extend($.fn.bootstrapTable.defaults.icons, {
+    autoRefresh: 'glyphicon-time icon-time'
+  });
+
+  $.extend($.fn.bootstrapTable.locales, {
+    formatAutoRefresh: function() {
+      return 'Auto Refresh';
+    }
+  });
+
+  var BootstrapTable = $.fn.bootstrapTable.Constructor;
+  var _init = BootstrapTable.prototype.init;
+  var _initToolbar = BootstrapTable.prototype.initToolbar;
+  var sprintf = $.fn.bootstrapTable.utils.sprintf;
+
+  BootstrapTable.prototype.init = function () {
+    _init.apply(this, Array.prototype.slice.apply(arguments));
+
+    if (this.options.autoRefresh && this.options.autoRefreshStatus) {
+      var _this = this;
```

Comment:
Instead of use **_this** use **that**.. this is because we always use **that** when we need to keep the **this** reference